### ISSUE TITLE
:focus-visible prefixed to :-moz-focusring ? 

### ISSUE THREAD

**Issue by rndmerle** [postcss/autoprefixer#1188]
Hi.

Any intent to support the prefixing of :focus-visible to :-moz-focusring ?
I'm not talking about polyfilling, just about the Firefox's prefix.

https://caniuse.com/#search=%3Afocus-visible

**Comment by ai** [comment]
Good idea. I will investigate it on next week. You can help if by links on research.

**Comment by rndmerle** [comment]
Hmm, I think I jumped the gun on this.
I played a bit more with `:-moz-focusring` and it appears it's not really following `:focus-visible` [specs](https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo).

They're talking about that here actually: https://bugzilla.mozilla.org/show_bug.cgi?id=1445482
and a [Codepen ](https://codepen.io/fvsch/pen/KGLZBa) shows that `:-moz-focusring` is about the same than `:focus`
I'm quite disappointed.

On the other hand `:focus-visible` is behind a flag on Chrome 67 without prefixing.
So I guess this pseudo-class is not autoprefixer's material after all :(
